Docker-Swarm-deployment/single-node/ogc-rs/ogc-rs-stack.yaml

```diff
@@ -0,0 +1,49 @@
+version: "3.9"
+
+services:
+  ogc-rs:
+    image: ghcr.io/datakaveri/geoserver-dev:1.0.0-alpha-96bb4be
+    command: bash -c "exec java $$OGCRS_JAVA_OPTS  -Dvertx.logger-delegate-factory-class-name=io.vertx.core.logging.Log4j2LogDelegateFactory -jar ./fatjar.jar  --host $$(hostname) -c secrets/configs/config.json"
+    cap_drop:
+      - ALL
+    read_only: true
+    env_file:
+      - ./secrets/.ogc-rs.env
+    volumes:
+      - type: volume
+        source: ogc-rs-volume
```

Comment:
Is this volume needed? 
cc + @karun-singh

Reply:
@abhi4578 @sivanaikk No, this is not needed. Can be removed.

Reply:
Updated the stack file